inc/class-antivirus.php

```diff
@@ -363,10 +373,23 @@ private static function _get_current_theme() {
 			return false;
 		}
 
+		// Append parent's data, if we got a child theme.
+		if ( false !== $theme->parent() ) {
+			$parent = self::_get_current_theme( $theme->parent() );
+		} else {
```

Comment:
The else is not really necessary because it is possible to set the bar before the if check.

Reply:
You're talking about default assignment like
```php
$parent = false;
if ( false !== $theme->parent() ) {
	$parent = self::_get_current_theme( $theme->parent() );
}
```
right?

Statement's too long for readably ternary operator imho.
```php
$parent = ( false !== $theme->parent() ) ? self::_get_current_theme( $theme->parent() ) : false;
```

I personally thik of the /short) if-else block as semantically more speaking than default assignment. However no strong preference here. Result is the same.

Reply:
No, I mean more simpler.

```php
// Append parent's data, if we got a child theme.
$parent = false;
if ( false !== $theme->parent() ) {
	$parent = self::_get_current_theme( $theme->parent() );
}
```

So that we not use an else, too much effort.

Reply:
That’s exactly what I wrote and again, I strongly prefer the else notation over default assignment, as it is more speaking. Can’t see any “effort“ on that one-line if-else...

Reply:
The one liner is okay, but not fine for the wp codex. However I mean the else is not necessary. On default you set to false and after the if statement is true, you change the var. Ist easier to read and faster on the server.

Reply:
I understand your point, but I got different opinion with different arguments. It’s either slower (not worth the discussion though) or the same in this case.

Any refs on how on the WP Codex argument?